src/styles/main.scss

```diff
@@ -0,0 +1,58 @@
+@import '../styles/variables';
+
+body {
+  margin: 0;
+  width: 100vw;
```

Comment:
Avoid using 100vw for widths because it can cause issues with scrollbars on some browsers. Instead, use 100% to ensure the body takes the full width of the viewport without causing horizontal scroll.